src/resources/views/crud/columns/week.blade.php

```diff
@@ -0,0 +1,6 @@
+@php
+    $column['prefix'] = $column['prefix'] ?? 'Week ';

```

Comment:
```suggestion
    $column['prefix'] = $column['prefix'] ?? __('backpack.???.Week').' ';
```

Week should be translated